components/eamxx/src/dynamics/homme/interface/dyn_grid_mod.F90

```diff
@@ -87,8 +85,9 @@ subroutine get_my_dyn_data (dg_gids, cg_gids, elgpgp, elgids, lat, lon)
       do ip=1,np
         do jp=1,np
           idof = (ie-1)*16+(jp-1)*4+ip
-          cg_gids(idof) = INT(el_cg_gids(ip,jp,ie),kind=c_int)
-          dg_gids(idof) = INT(el_dg_gids(ip,jp,ie),kind=c_int)
+          ! Fortran GIDs are 1-based, but we want 0-based ids
```

Comment:
Because this touches homme code, we will need to go the regular `next` -> `master` route.  Just noting here since it could be missed at integration.

Reply:
This is not Homme, it's still eamxx. It's the _eamxx interface to homme_.